commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/Drive.scala

```diff
@@ -9,8 +9,8 @@ trait Drive {
   type DriveSignal
   type DriveVelocity
 
-  protected def driveClosedLoop[C[_]](signal: SignalLike[DriveSignal, C]): PeriodicSignal[DriveSignal]
-  protected def driveVelocity[C[_]](velocity: SignalLike[DriveVelocity, C]): PeriodicSignal[DriveSignal]
+  protected def driveClosedLoop(signal: SignalLike[DriveSignal]): PeriodicSignal[DriveSignal]
+  protected def driveVelocity(velocity: SignalLike[DriveVelocity]): PeriodicSignal[DriveSignal]
```

Comment:
The first takes a target percentage (like what joysticks would have) and runs velocity control on that. The second takes a velocity (in proper velocity units) and controls to that. I really want to clean this up at some point, but have to figure out how to do that without forcing anyone who has non-velocity units signals to first convert them to velocity units before running velocity control.